content/docs/faq-functions.md

```diff
@@ -29,71 +29,71 @@ class Foo extends Component {
     this.handleClick = this.handleClick.bind(this);
   }
   handleClick() {
-    console.log('Click happened');
+    console.log('Clicado');
   }
   render() {
-    return <button onClick={this.handleClick}>Click Me</button>;
+    return <button onClick={this.handleClick}>Clique em mim!</button>;
```

Comment:
```suggestion
   return <button onClick={this.handleClick}>Clique Aqui!</button>;
```

Reply:
@glaucia86 eu acabei de atualizar o glossário, acho válido a tradução para blocos dentro de HTML/JSX visto que eles não mudam a estrutura/lógica e ajudam ao leitor entender o contexto. o que você acha?

Reply:
Show! Perfeito! Era só isso que faltava para aprovar esse PR?
Tou com dúvida ainda com respeito aos títulos se devem ser traduzidos ou não. Pois aquilo ali são Titulos. Uma vez que alguém for clicar no título e não estiver co-relacionado ao link... não vai funcionar. @cezaraugusto